appyters/RNA_Level_Functional_Predictions/static/TCGA.json

```diff
@@ -0,0 +1 @@
2", "ENSG00000235904", "ENSG00000235908", "ENSG00000235912", "ENSG00000235916", "ENSG00000235917", "ENSG00000235929", "ENSG00000235934", "ENSG00000235937", "ENSG00000235939", "ENSG00000235944", "ENSG00000235945", "ENSG00000235946", "ENSG00000235949", "ENSG00000235957", "ENSG00000235967", "ENSG00000235974", "ENSG00000235978", "ENSG00000235979", "ENSG00000235982", "ENSG00000235992", "ENSG00000235996", "ENSG00000235997", "ENSG00000236003", "ENSG00000236005", "ENSG00000236015", "ENSG00000236017", "ENSG00000236018", "ENSG00000236021", "ENSG00000236028", "ENSG00000236030", "ENSG00000236031", "ENSG00000236035", "ENSG00000236044", "ENSG00000236045", "ENSG00000236053", "ENSG00000236054", "ENSG00000236056", "ENSG00000236060", "ENSG00000236065", "ENSG00000236066", "ENSG00000236069", "ENSG00000236073", "ENSG00000236086", "ENSG00000236090", "ENSG00000236095", "ENSG00000236098", "ENSG00000236115", "ENSG00000236116", "ENSG00000236118", "ENSG00000236124", "ENSG00000236132", "ENSG00000236136", "ENSG00000236140", "ENSG00000236146", "ENSG00000236152", "ENSG00000236153", "ENSG00000236154", "ENSG00000236164", "ENSG00000236165", "ENSG00000236166", "ENSG00000236170", "ENSG00000236173", "ENSG00000236182", "ENSG00000236184", "ENSG00000236187", "ENSG00000236190", "ENSG00000236194", "ENSG00000236197", "ENSG00000236199", "ENSG00000236200", "ENSG00000236208", "ENSG00000236209", "ENSG00000236212", "ENSG00000236213", "ENSG00000236216", "ENSG00000236229", "ENSG00000236230", "ENSG00000236231", "ENSG00000236234", "ENSG00000236255", "ENSG00000236257", "ENSG00000236259", "ENSG00000236262", "ENSG00000236264", "ENSG00000236266", "ENSG00000236267", "ENSG00000236269", "ENSG00000236274", "ENSG00000236278", "ENSG00000236281", "ENSG00000236283", "ENSG00000236285", "ENSG00000236304", "ENSG00000236307", "ENSG00000236308", "ENSG00000236312", "ENSG00000236318", "ENSG00000236323", "ENSG00000236324", "ENSG00000236325", "ENSG00000236327", "ENSG00000236333", "ENSG00000236337", "ENSG00000236341", "ENSG00000236347", "ENSG00000236349", "ENSG00000236352", "ENSG00000236358", "ENSG00000236360", "ENSG00000236364", "ENSG00000236375", "ENSG00000236377", "ENSG00000236380", "ENSG00000236385", "ENSG00000236388", "ENSG00000236389", "ENSG00000236390", "ENSG00000236391", "ENSG00000236394", "ENSG00000236397", "ENSG00000236409", "ENSG00000236411", "ENSG00000236417", "ENSG00000236420", "ENSG00000236426", "ENSG00000236427", "ENSG00000236430", "ENSG00000236431", "ENSG00000236434", "ENSG00000236439", "ENSG00000236447", "ENSG00000236449", "ENSG00000236451", "ENSG00000236452", "ENSG00000236453", "ENSG00000236456", "ENSG00000236457", "ENSG00000236459", "ENSG00000236463", "ENSG00000236469", "ENSG00000236471", "ENSG00000236472", "ENSG00000236473", "ENSG00000236474", "ENSG00000236478", "ENSG00000236480", "ENSG00000236481", "ENSG00000236485", "ENSG00000236491", "ENSG00000236493", "ENSG00000236494", "ENSG00000236498", "ENSG00000236501", "ENSG00000236508", "ENSG00000236510", "ENSG00000236514", "ENSG00000236516", "ENSG00000236
```

Comment:
Can you investigate whether such a list already exists somewhere in a public location such that it could be fetched from there?

Reply:
Hi Daniel,
The list exists in the AWS bucket in minio. However, I'm not sure how to use s3fs in JavaScript. I will investigate ways to access the HDF file though

Reply:
Please note that the s3 protocol is defined to *also function over http*. In most cases this can be achieved with:
`s3://yourbucket.s3.amazonaws.com/your_file => https://yourbucket.s3.amazonaws.com/your_file`